src/extension/background-script/events/notifications.js

```diff
@@ -6,11 +6,18 @@ const paymentSuccessNotification = (message, data) => {
   const recipient = data.origin.name;
   const paymentResponse = data.response;
   const route = paymentResponse.data.route;
-  const { total_amt } = route;
+  const { total_amt, total_fees } = route;
+  const paymentAmount = total_amt - total_fees;
+
+  function formatAmount(amount) {
+    return `${amount} sat${amount != 1 ? "s" : ""}`;
+  }
 
   return utils.notify({
-    title: `Paid ${total_amt} sat to ${recipient}`,
-    message: `pre image: ${paymentResponse.data.preimage}`,
+    title: `✅ Successfully paid ${formatAmount(
+      paymentAmount
+    )} to »${recipient}«`,
```

Comment:
The `recipient` is more like the `initiator` of the payment, right? We don't know who the user actually just paid 🤔

Reply:
I mean it could be "paid X ON foo.com" - if that's better?

Reply:
Generally I like ON very much because that describes exactly what actually happened. 

But that wouldn't work for LightningAddress payments directly in Alby. There the success notification now says `Successfully paid X to example@example.com` which is a good thing IMO. We would need to come up with one phrase that fits both use cases or be able to distinguish the 2 cases:

 - You have successfully paid X on example.com (calls through webln)
  - You have successfully paid X to lnaddress@example.com (payments initiated by alby batteries, send tab, etc)

Reply:
After having a look at how this is handled internally I think this is a whole new topic that goes way beyond this notification: 

Currently Alby does not differentiate the `initiator` or `payee`. If you for instance pay an invoice the UI & notifications indicate that you are currently paying Alby. (as this is probably the default?)

I'd create a separate issue for that and discuss it in there and leave the notification text as it is now.

Reply:
> If you for instance pay an invoice the UI & notifications indicate that you are currently paying Alby. (as this is probably the default?)

this was because of pragmatic reasons to reuse what was there. 

+1 for a new issue to look into this as a whole. thx.

Reply:
I've created a follow-up issue and tried to summarize my findings:
https://github.com/getAlby/lightning-browser-extension/issues/770

Please let me know if I missed anything.